Makefile

```diff
@@ -88,6 +95,25 @@ build: build-agent build-server build-cli
 
 release-frontend: build-frontend
 
+check-xgo:
+	@hash xgo > /dev/null 2>&1; if [ $$? -ne 0 ]; then \
+		$(GO) install src.techknowlogick.com/xgo@latest; \
+	fi
+
+cross-compile-server: cross-compile-server-build-loop
+	tree dist
```

Comment:
Hi, could you just run `tree` to see if the binaries are placed outside of the `dist/` folder?

Reply:
Binaries are placed in `/build/`